util/execdetails/execdetails.go

```diff
@@ -141,6 +161,46 @@ func (e *RuntimeStatsColl) Get(planID string) *RuntimeStats {
 	return runtimeStats
 }
 
+// RecordOneCopTask records a specific cop tasks's execution detail.
+func (e *RuntimeStatsColl) RecordOneCopTask(planID, address string,
+	timeProcessedNs, numProducedRows, numIterations uint64) {
+	e.mu.Lock()
+	defer e.mu.Unlock()
+
+	_, ok := e.copStats[planID]
+	if !ok {
+		e.copStats[planID] = make(map[string][]*CopRuntimeStats, 8)
+	}
+
+	e.copStats[planID][address] = append(e.copStats[planID][address],
+		&CopRuntimeStats{timeProcessedNs, numProducedRows, numIterations})
+}
+
+// CopSummary gets a summary of the cop task's execution information.
+func (e *RuntimeStatsColl) CopSummary(planID string) string {
+	e.mu.Lock()
+	defer e.mu.Unlock()
+	stats, ok := e.copStats[planID]
+	if !ok {
+		return ""
+	}
+	var totalRows, totalIters, totalTasks uint64
+	procTimes := make([]uint64, 0, 32)
+	for _, instanceStats := range stats {
+		for _, stat := range instanceStats {
+			procTimes = append(procTimes, stat.timeProcessedNs)
+			totalRows += stat.numProducedRows
+			totalIters += stat.numIterations
+			totalTasks++
+		}
+	}
+
+	n := len(procTimes)
+	sort.Slice(procTimes, func(i, j int) bool { return procTimes[i] < procTimes[j] })
+	return fmt.Sprintf("procNs max:%v, min:%v, p80:%v, p95:%v, rows:%v, iters:%v, tasks:%v",
```

Comment:
Can we also add the unit for time? It is hard to interpret something like `procNs max:129931, min:129931, p80:129931, p95:129931, rows:2, iters:2, tasks:1`.